### PR TITLE
Add missing experimental decorators to the comet integration

### PR DESCRIPTION
**Pull request by nzw0301**
## Motivation

Adding `experimental` decorators will make the class and method more user-friendly by following the convention of other integration classes and methods. 

## Description of the changes

See title.

### CODE REVIEW

File: optuna_integration/comet/comet.py [hunk 1]
```diff
@@ -213,6 +216,7 @@ def _init_optuna_trial_experiment(
         setattr(comet_ml, "active_experiment", experiment)
         return experiment
 
+    @experimental_func("4.0.0")
```
Comment (not522) [inline optuna_integration/comet/comet.py:219]: Since the entire class is experimental, isn't a decorator unnecessary for the method?

Comment (nzw0301) [reply optuna_integration/comet/comet.py:219]: Thanks. You're right. I just followed similar implementations, such as https://github.com/optuna/optuna-integration/blob/605390c85f86f7eef621e6339dd9fcd41db26611/optuna_integration/wandb/wandb.py#L187, but this looks unnecessary too.

Comment (nzw0301) [inline optuna_integration/comet/comet.py:219]:
```suggestion
```

File: optuna_integration/comet/comet.py [hunk 2]
```diff
@@ -6,6 +6,8 @@
 import json
 
 import optuna
+from optuna._experimental import experimental_class
+from optuna._experimental import experimental_func
```
Comment (not522) [inline optuna_integration/comet/comet.py:10]: Could you remove this import?
```suggestion
```

**Review by not522 — APPROVED** [review]
LGTM!

**Pull request merged**